frontend/packages/console-app/src/components/nodes/node-dashboard/NodeAlerts.tsx

```diff
@@ -184,8 +186,8 @@ const NodeAlerts: React.FC = ({ children }) => {
             totalKey={NodeQueries.CPU_TOTAL}
             limitKey={NodeQueries.POD_RESOURCE_LIMIT_CPU}
             requestedKey={NodeQueries.POD_RESOURCE_REQUEST_CPU}
-            limitState={cpuLimit?.limit}
-            requestedState={cpuLimit?.requested}
+            limitState={cpuLimit?.limit || LIMIT_STATE.OK}
+            requestedState={cpuLimit?.requested || LIMIT_STATE.OK}
```

Comment:
i am worried that this may cause nodes to state OK when the lack of a limit state may indicate something else